core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -51,11 +51,15 @@ set_error_buf_v(char *error_buf, uint32 error_buf_size, const char *format, ...)
 }
 
 WASMModule *
-wasm_load(uint8 *buf, uint32 size, char *error_buf, uint32 error_buf_size)
+wasm_load(uint8 *buf, uint32 size,
+#if WASM_ENABLE_MULTI_MODULE != 0
+          bool main_module,
```

Comment:
not sure about it. `wasm_runtime_load` is the only caller of `wasm_load` and we don't want API users to load a sub-module manually and seems we don't need to add `main_module` flag here because it is always "true".